.changeset/cold-news-glow.md

```diff
@@ -0,0 +1,14 @@
+---
+"@localyze-pluto/components": major
+---
+
+[ContentCard]
+
+> Breaking change: `iconUrl` prop is now `icon` and accepts a ReactNode
```

Comment:
Considering that it accepts ReactNode (which could be anything, not just an icon or an image), and that you are also adding topRightElement, why not call this topLeftElement?

Reply:
Because it's only an icon for the library, for Figma. We don't want to have a button now, for example, and I didn't want to restrict the type of the icon to string or Images from NextJS because for me it would be too specific.

Reply:
I agree it would be too specific, but the name of the prop doesn't really match what the prop is for. Even if we don't want to have a button now, it is possible to do it. And considering that this component lives in pluto (DS) and not in the app, I would argue it doesn't really matter "how" we want to use it, but rather what the component is capable of

Reply:
Changed this here https://github.com/Localitos/pluto/pull/1752/files#diff-f7fa8c6df160defa8d1aad22a1a3e5af7da525824cca21b138dee5363ceedbd9R56